components/compliance-service/ingest/ingestic/ingestic.go

```diff
@@ -354,12 +362,17 @@ func (backend *ESClient) setYesterdayLatestToFalse(ctx context.Context, nodeId s
 		logrus.Debugf("setYesterdayLatestToFalse: updating day_latest=false on %s", indexOneDayAgo)
 	}
 
-	_, err := elastic.NewUpdateByQueryService(backend.client).
-		Index(indexOneDayAgo + "*").
-		Query(boolQueryDayLatestThisNodeNotThisReport).
-		Script(script).
-		Refresh("false").
-		Do(ctx)
+	// If the date of index isn't the current date then update  the changes
+	for _, ind := range indexes {
```

Comment:
instead of looping and modifying each index one at a time, is there any way to update all the last 90 days' indexes in one go?